### PR TITLE
Fix/swerve snap clean

### PR DESCRIPTION
**Pull request by LucienMorey**
just for you davo <3

### CODE REVIEW

File: components/chassis.py [hunk 1]
```diff
@@ -94,15 +92,11 @@ def get_speed(self):
     def set(self, desired_state: SwerveModuleState):
 
         if abs(desired_state.speed) < 1e-3:
-            if abs(self.last_speed) > 1e-13:
-                self.drive.setIntegralAccumulator(0, 0, 0)
-                self.drive.set(ctre.ControlMode.Velocity, 0)
-                self.steer.set(ctre.ControlMode.Velocity, 0)
-                self.last_speed = 0
+            self.drive.set(ctre.ControlMode.Velocity, 0)
+            self.steer.set(ctre.ControlMode.Velocity, 0)
```
Comment (auscompgeek) [inline components/chassis.py:96]: I do wonder if setting steer to velocity of 0 is a good idea if we normally run it on position PID...

File: components/chassis.py [hunk 2]
```diff
@@ -94,15 +92,11 @@ def get_speed(self):
...
+            self.last_speed = 0
```
Comment (auscompgeek) [inline components/chassis.py:97]:
```suggestion
```

**Review by auscompgeek — APPROVED** [review]

File: components/chassis.py (hunk 1 above)
Comment (LucienMorey) [reply components/chassis.py:96]: I dont think it really matters for now since things are working fine. This is the easiest fix for a pretty loud bug lmao

Comment (auscompgeek) [reply components/chassis.py:96]: Well, if you say it works fine, then :rocket:

**Comment by auscompgeek** [comment]
ping @LucienMorey

**Pull request merged**